openpype/plugins/publish/integrate.py

```diff
@@ -891,3 +909,34 @@ def prepare_file_info(self, path, anatomy, sites):
             "hash": source_hash(path),
             "sites": sites
         }
+
+    def _is_path_in_project_roots(self, roots, file_path):
+        """Checks if 'file_path' starts with any of the roots.
+
+        Used to check that published path belongs to project, eg. we are not
+        trying to publish to local only folder.
+        Args:
+            roots (list of RootItem): {ROOT_NAME: ROOT_PATH}
+            file_path (str)
+        Returns:
+            (bool)
+        """
+        file_path = str(file_path).replace("\\", "/").lower()
```

Comment:
Please use `anatomy.find_root_template_from_path` to find the root (if the logic won't change).

Reply:
The issue is that in that method comparison is case sensitive, even if it doesn't need to be (on Windows).
But I am bit squeamish about slap there `lower` (because of Linux).
![find_root_template_from_path_cleanup](https://user-images.githubusercontent.com/4457962/206178510-60a06afb-a054-4a5d-a841-14d839da5e96.png)

Reply:
Well then we should modify the `find_root_template_from_path` to don't care about case sensitivity on windows?

Reply:
Updated `find_root_template_from_path `